src/core/raster/qgshillshaderenderer.cpp

```diff
@@ -264,6 +304,13 @@ QgsRasterBlock *QgsHillshadeRenderer::block( int bandNo, const QgsRectangle &ext
     }
   }
 
+#ifdef QGISDEBUG
+  QgsMessageLog::logMessage( QStringLiteral( "CPU Rendering time for hillshade (%2 x %3 ): %4 ms" )
```

Comment:
This should probably also depend on the rendering debug option being enabled.

Reply:
@m-kuhn do you mean `settings.value( QStringLiteral( "Map/logCanvasRefreshEvent" ), false ).toBool()` ?